scss/components/table.scss

```diff
@@ -166,19 +181,18 @@ $block: #{$fd-namespace}-table;
         }
     }
 
-    th.fd-table__context-menu {
+    .#{$block}__header .#{$block}__cell.fd-table__context-menu {
```

Comment:
Couple things:

First, for better readability and searching, let's change this to `.#{$block}__contextual-menu`.

Second, can we write this in better sass syntax with nesting for descendant and sibling classes?